more/free-programming-cheatsheets.md

```diff
@@ -80,6 +80,7 @@
 ### <a name="ide--editores"></a>IDE / Editors
 
 * [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [GNU Emacs Refcard](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - Official PDF from [GNU.org](https://www.gnu.org/home.en.html)
```

Comment:
Use this snippet as fix. Ot should solve linter faults :wink:

- PDF links should have `(PDF)` format mark

```markdown
* [GNU Emacs Reference Card](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - GNU.org (PDF)
```